src/pymmcore_widgets/_shutter_widget.py

```diff
@@ -280,16 +278,17 @@ def _on_prop_changed(self, dev_name: str, prop_name: str, value: Any) -> None:
                 if value != "Undefined":
                     self._mmc.events.propertyChanged.emit(value, "State", True)
 
+    def _on_live_mode(self) -> None:
+        if not self._mmc.getShutterOpen(self.shutter_device):
+            self._set_shutter_wdg_to_closed()
+
```

Comment:
@fdrgsp note that I added in the check you suggested whenever on sequence acquiistion starts or stops.

Reply:
ah, and also see that it's not complete....

Reply:
now fixed for real